### PR TITLE
Add post announcing the Storage Bucket OT

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add post announcing the Storage Bucket OT.

### CODE REVIEW

File: site/en/blog/storage-buckets-origin-trial/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+hero: 'image/8WbTDNrhLsU0El80frMBGE4eMCD3/uSUKNoGdlmEsXAtHr0yJ.jpg'
+alt: >
+  Several buckets with paint in different colors in them.
+---
+
+The [Storage Standard](https://storage.spec.whatwg.org/) defines an API for persistent storage and
+quota estimates, and the platform storage architecture.
+We're experimenting with an API for making persistent storage eviction under heavy
+memory pressure more predictable.
+Try the Storage Buckets API by [participating in the Storage Buckets origin trial](https://developer.chrome.com/origintrials/#/view_trial/2674012278751232001). The origin trial runs from Chrome&nbsp;115 (stable date): {% ChromeDate 115 %} to Chrome&nbsp;118 (stable date): {% ChromeDate 118 %}. For all details on the Storage Bucket API, see the article [Not all storage is created equal: introducing Storage Buckets](/blog/storage-buckets/).
```
Comment (tomayac) [inline site/en/blog/storage-buckets-origin-trial/index.md:23]:
```suggestion
Try the Storage Buckets API by [participating in the Storage Buckets origin trial](/origintrials/#/view_trial/2674012278751232001). The origin trial runs from Chrome&nbsp;115 (stable date): {% ChromeDate 115 %} to Chrome&nbsp;118 (stable date): {% ChromeDate 118 %}. For all details on the Storage Bucket API, see the article [Not all storage is created equal: introducing Storage Buckets](/blog/storage-buckets/).
```

File: site/en/blog/storage-buckets-origin-trial/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Storage Buckets origin trial available
+description: >
+  The Storage Standard defines an API for persistent storage and quota estimates, and the platform storage architecture. We're experimenting with an API for making persistent storage eviction under heavy memory pressure more predictable. Try the Storage Buckets API in an origin trial that runs from Chrome 115 to Chrome 118.
+subhead: >
+  Optimize persistent storage eviction under heavy memory pressure.
+date: 2021-05-15
```
Comment (rachelandrew) [inline site/en/blog/storage-buckets-origin-trial/index.md:8]: This date appears to be wrong.

File: site/en/blog/storage-buckets-origin-trial/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+hero: 'image/8WbTDNrhLsU0El80frMBGE4eMCD3/uSUKNoGdlmEsXAtHr0yJ.jpg'
+alt: >
+  Several buckets with paint in different colors in them.
+---
+
+The [Storage Standard](https://storage.spec.whatwg.org/) defines an API for persistent storage and
+quota estimates, and the platform storage architecture.
+We're experimenting with an API for making persistent storage eviction under heavy
+memory pressure more predictable.
+Try the Storage Buckets API by [participating in the Storage Buckets origin trial](/origintrials/#/view_trial/2674012278751232001). The origin trial runs from Chrome&nbsp;115 (stable date): {% ChromeDate 115 %} to Chrome&nbsp;118 (stable date): {% ChromeDate 118 %}. For all details on the Storage Bucket API, see the article [Not all storage is created equal: introducing Storage Buckets](/blog/storage-buckets/).
```
Comment (rachelandrew) [inline site/en/blog/storage-buckets-origin-trial/index.md:23]:
```suggestion
The experimental [Storage Buckets API](/blog/storage-buckets/) is our attempt to make persistent storage eviction under heavy memory pressure more predictable.

Try the Storage Buckets API by [participating in the Storage Buckets origin trial](/origintrials/#/view_trial/2674012278751232001). The origin trial runs from Chrome&nbsp;115 (stable date): {% ChromeDate 115 %} to Chrome&nbsp;118 (stable date): {% ChromeDate 118 %}.

**Review by rachelandrew — CHANGES_REQUESTED** [review]
I don't think this is tracked?

File: site/en/blog/storage-buckets-origin-trial/index.md (hunk 2 above)
Comment (tomayac) [inline site/en/blog/storage-buckets-origin-trial/index.md:8]:
```suggestion
date: 2023-05-15
```

**Comment by tomayac** [comment]
Tracker entry added.

**Pull request merged**